docs/tools/cli/service.rst

```diff
@@ -96,6 +96,8 @@ Creates a new service.
     - Aiven subscription plan name; check :ref:`avn_service_plan` for more information
   * - ``--cloud``
     - The cloud region name; check :ref:`avn-cloud-list` for more information
+  * - ``--disk-space-gib``
+    - Extra amount of disk space for data storage (GiB)
```

Comment:
This should be the total amount of disk space, not the extra amount

Reply:
Well, not always total - it will be total for Kafka because the customer controls the replica numbers, but for (e.g.) postgresql where it's primary + replicas then I think it's the per-node disk space, effectively the total usable space.